src/Giraffe/ResponseWriters.fs

```diff
@@ -9,16 +10,47 @@ open Microsoft.Net.Http.Headers
 open FSharp.Control.Tasks.V2.ContextInsensitive
 open Giraffe.GiraffeViewEngine
 
+
 // ---------------------------
 // Helper functions
 // ---------------------------
 
+let private DefaultCapacity = 1386
+let private MaxBuilderSize  = DefaultCapacity * 3
```

Comment:
@dv00d00 Would like to discuss these limits. How did you conclude `1386` as the default size and 3 times of it as the max size?